spec/classes/nginx_spec.rb

```diff
@@ -1124,7 +1146,9 @@
 
                   # if we have a _path attribute make sure we create the path
                   if param[:attr].end_with?('_path')
-                    if param[:value].is_a?(Hash)
+                    if (param[:attr] == 'client_body_temp_path' || param[:attr] == 'proxy_temp_path') && param[:value].is_a?(Array)
```

Comment:
I'd write this as
```suggestion
                    if ['client_body_temp_path', 'proxy_temp_path'].include?(param[:attr]) && param[:value].is_a?(Array)
```

Slight nit: to keep the diff smaller I'd normally write it as `elsif`. Now you're also changing the `if` itself which makes the diff larger.